src/Client.js

```diff
@@ -16,10 +17,103 @@ var values = require('./values')
  * @param {RequestResult} res
  */
 
+/**
+ * **WARNING: This is an experimental feature. There are no guarantees to
+ * its API stability and/or service availability. DO NOT USE IT IN
+ * PRODUCTION**.
+ *
+ * Creates a subscription to the result of the given read-only expression. When
+ * executed, the expression must only perform reads and produce a single
+ * streamable type, such as a reference or a version. Expressions that attempt
```

Comment:
what `version` means in that context? the same as in core?

Reply:
Yes. It means that both `Ref(..)` and `Get(Ref(..))` are streamable types.